examples/test-size5-no924 2.newick

```diff
@@ -0,0 +1,8 @@
+(m1,((m4,m5)m3,(m7,m8)m6)m2)m0;
```

Comment:
Duplicate -- it seems like these duplicates might have been created my merge? Please be a little careful when you merge.

Reply:
Removed.